server_apps/data_transfer/Downloads/GFF3/knockdown_reagents/load_knockdown_reagents.sh

```diff
@@ -5,7 +5,7 @@ TARGETDIR="$TARGETROOT/server_apps/data_transfer/Downloads/GFF3/knockdown_reagen
 
```

Comment:
I noticed this file uses `/bin/sh`. I wonder if we should be explicit about the shell and change it to /bin/bash? If we do, we can use `set -e` which will stop the script if it encounters any errors.

Reply:
Sounds good.